api/swagger/swagger-v1-full.yaml

```diff
@@ -293,6 +293,53 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/coin_members_count_response'
+  /coins/volume-leaders:
```

Comment:
FWIW I've been skipping the "full" documentation if there's no difference between the non-full and full endpoints - how do we feel about that? My feeling is that it's easier to maintain and we're hopefully moving away from "full" as we go on

EDIT: Spoke too soon, I see "full" has "full users"

Reply:
Yeah normally I wouldn't bother, but I actually think for the leaderboard we probably want to use the "min" endpoint since we don't need all the extra info. So figured might as well generate both